### ISSUE TITLE
License ?

### ISSUE THREAD

**Issue by Poussinou** [Humzaman/removed#1]
Hi,

What's the license of this project?

If it's something open-source like the GNU GPL v3 license, we could try to add it to F-Droid later, if you're ok.

**Comment by humzakh** [comment]
Hey bud,

Just added GNU GPL v3 license. Thanks for reminding me.

I tried to add the app to F-Droid a while back but kept running into this error when working with fdroid-server: 
`CRITICAL: 'MTuf45ZvxhMWWo4v8YBbFDTLsFcGtpcPNT' is not a valid Litecoin address in com.readrops.app. Regex pattern: ^L[a-zA-Z0-9]{33}$`

Didn't get around to figuring out the problem

**Comment by Poussinou** [comment]
@Izzysoft any idea here for this problem? I see there are 34 symbols in the adress

**Comment by IzzySoft** [comment]
@Poussinou @Humzaman Yes. Use `fdroidserver` cloned from the git repo, and make sure your fork of `fdroiddata` is up-to-date. That's an old issue with the `fdroidserver` coming via the OS repos (Debian etc) which has long been fixed.

**Comment by Kelvino9** [comment]
@Humzaman any progress on solving the F-Droid issues?
